.travis.yml

```diff
@@ -10,6 +10,9 @@ branches:
 before_install:
 - npm config set spin false
 - npm install -g bower coveralls pr-bumper
+- bower --version
+- npm install phantomjs-prebuilt
```

Comment:
For what it's worth, I lost half a day of productivity trying to track down some hard to decipher bugs due to lack of phantom support.